tests/test_georaster.py

```diff
@@ -704,6 +711,20 @@ def test_crop(self, data: list[str]) -> None:
         r_cropped2 = r.crop(r_cropped, inplace=False)
         assert r_cropped2 == r_cropped
 
+        # Check that bound reprojection is done automatically if the CRS differ
+        with warnings.catch_warnings():
```

Comment:
Same as below; why do we expect warnings here?

Reply:
Done as above